### ISSUE TITLE
Add in README.md file. About injection of 'ionic.contrib.ui.hscrollcards' in the app module.

### ISSUE THREAD

**Issue by rajeshwarpatlolla** [drewrygh/ionic-ion-horizontal-scroll-cards#3]
I was using this component, i tried few times with the description you mentioned there. But i was unable to see any thing in the page. After injecting your module name in the main module of my application, i got the images sliding in the horizontal direction. Please mention that we have to inject `ionic.contrib.ui.hscrollcards` in the main module in your README.md file.

**Comment by hasokeric** [comment]
Yep - Its not in the instructions but i looked at the sample code provided and saw it there :-) +1

**Comment by majornorth** [comment]
+1.

I figured this out from a read me for the Tinder plugin, but even after injecting, I still can't get it to work.

**Comment by payas** [comment]
ufff, you save me
